cairo-prover-entrypoint.sh

```diff
@@ -3,19 +3,20 @@
 # Read from stdin
 cat > input.json && \
 
+layout=$(jq -r '.layout' input.json) && \
 jq -r '.program_input | join(" ")' input.json | tr -d '\n' > program_input.txt && \
 jq '.program' input.json > program.sierra.json && \
 
 cairo1-run \
     --trace_file program_trace.trace \
     --memory_file program_memory.memory \
-    --layout recursive \
+    --layout $layout \
     --proof_mode \
     --air_public_input program_public_input.json \
     --air_private_input program_private_input.json \
     --args_file program_input.txt \
     program.sierra.json \
-    2>&1 > /dev/null && \
+    2>&1 > /dev/null 
```

Comment:
leave `&& \` not to execute next line if failed

Reply:
done